opds_import.py

```diff
@@ -929,7 +929,6 @@ def _data_detail_for_feedparser_entry(cls, entry, metadata_data_source):
                         )
                     )
         last_opds_update = cls._datetime(entry, 'updated_parsed')
-        added_to_collection_time = cls._datetime(entry, 'published_parsed')
```

Comment:
Here's a line where we were getting the conflated atom:published/dc:issued value, but we were never using it.